SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -516,5 +517,10 @@ bool SpatialGDKGenerateSchema()
 
 	AdditionalSchemaGeneratedClasses.Empty();
 
+	if (bHasByRefRPCArgs)
+	{
+		UE_LOG(LogSpatialGDKSchemaGenerator, Warning, TEXT("Pass-by-reference arguments detected in some RPCs. Please check previous log messages for more details."));
```

Comment:
Is this repetition needed?

Reply:
Nice to have it at the bottom of the log.

Reply:
This log will be shown at the end of schema gen, so the previous warnings don't get accidentally ignored (e.g. if there are a lot of logs)